docs/project-structure.md

```diff
@@ -4,14 +4,23 @@ The directory layout of **Predictive Calc** is organized in a way that separates
 
 ```
 predictive-calc/
-├── app.py                          # Main entry point for the Streamlit web app
-├── docs/                           # Documentation files
-├── form_configs/                   # Configuration files for the forms
+├── app.py                           # Main entry point for the Streamlit web app
+├── docs/                            # Documentation files
+│   ├── project-structure.md         # Directory layout of the repository
+│   ├── tutorial.md                  # Steps to integrate a new machine learning model into the repository 
+├── form_configs/                    # Configuration files for the forms
+│   ├── customer_income.json         # JSON configuration for customer income model form input fields
+│   ├── gold_price_prediction.json   # JSON configuration for gold price prediction model form input fields
 │   ├── house_price.json             # JSON configuration for house price model form input fields
 │   ├── loan_eligibility.json        # JSON configuration for loan eligibility model form input fields
-├── models/                         # Folder containing machine learning models
-│   ├── house_price/                # Example model directory (for house price prediction)
-│   │   ├── data/                   # Datasets used by the models
+│   ├── parkinson_detection.json     # JSON configuration for parkinson disease detection model form input fields
+│   ├── stress_detection.json        # JSON configuration for stress detection model form input fields
+├── models/                          # Folder containing machine learning models
+│   ├── PDF_malware_detection
+│   ├── customer_income
+│   ├── gold_price_prediction
```

Comment:
Remove this.
Remove the individual file names, keeping just one or two `form_configs/` file detail to explain the overall folder structure.

Instead you could put `....` to denote more similar files.